src/openai_cli/test_cli.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from click.testing import CliRunner
+from openai_cli.cli import cli
+
+class TestCLI(unittest.TestCase):
+    def setUp(self):
+        self.runner = CliRunner()
```

Comment:
I find it best to cripple the test harness so it can't make an actual API call. This way, even if something slips in any of the unit tests, it would just fail with network error instead of making network call.
Previously, I was setting  `_API_URL = "api_url"` for this purpose.